### PR TITLE
Fix #1337 -  Loan transactions RecyclerView height and margins refactored

### LINKED ISSUE #1337: Loan transactions Recycler view
**Summary:**

The loan transactions recyclerview does not cover the entire screen which ideally it is supposed to do . Currently it covers only the area required area to contain the items inside it. Hence swiping across them results in bad user experience

**Steps to reproduce:**
* Go to accounts -> Loan tab
* Select any disbursed loan
* Tap on Transactions

**Expected behaviour:**

Ideally the recycler view should have covered the entire remaining portion of the screen

**Device and Android version:**

Xiaomi PocoF1
Android 9
Stock Rom

### PR DESCRIPTION
**Pull request by jawidMuhammadi**
Fixes #1337 

Loan transactions RecyclerView height and margins refactored. Because it was not covering the entire screen which ideally it is supposed to do. Swiping across them results in bad user experience and also it was not according to material design guidelines.
**Before:**
Screenshot_2020-02-13-23-57-06
Screenshot_2020-02-13-23-57-13

**Now:**
Screenshot_2020-02-13-23-41-59
Screenshot_2020-02-13-23-45-04
#1337

### CODE REVIEW

File: app/src/main/java/org/mifos/mobile/ui/fragments/LoanApplicationFragment.java [hunk 1]
```diff
@@ -547,6 +547,8 @@ public void onItemSelected(AdapterView<?> parent, View view, int position, long
 
             case R.id.sp_loan_purpose:
                 if (loanTemplate.getLoanPurposeOptions() != null) {
+                    if (position == loanTemplate.getLoanPurposeOptions().size())
+                        position--;
```
Comment (naman14) [inline app/src/main/java/org/mifos/mobile/ui/fragments/LoanApplicationFragment.java]: can you explain why are we doing this?

File: build.gradle [hunk 2]
```diff
@@ -38,7 +38,7 @@ ext {
     buildToolsVersion = '28.0.3'
 
     // App dependencies
-    supportLibraryVersion = '1.1.0'
+    supportLibraryVersion = '1.0.0'
```
Comment (naman14) [inline build.gradle]: remove this change, keep separate PRs for each issue/change

File: app/src/main/java/org/mifos/mobile/ui/fragments/LoanApplicationFragment.java (hunk 1 above)
Comment (jawidMuhammadi) [reply app/src/main/java/org/mifos/mobile/ui/fragments/LoanApplicationFragment.java]: Actually this line of code was regarding to another issue that has been raised by me. During committing I have written the wrong issue number in the commit message. I did update the commit message but I don't know why the change didn't appear here.

File: build.gradle (hunk 2 above)
Comment (jawidMuhammadi) [reply build.gradle]: Ok I will. But if I revert this change, the recycler items in saving account, loan account and share account do not work. Before pushing my code I asked the community one of them advised to not remove this change.

File: app/src/main/java/org/mifos/mobile/ui/fragments/LoanApplicationFragment.java (hunk 1 above)
Comment (jawidMuhammadi) [reply app/src/main/java/org/mifos/mobile/ui/fragments/LoanApplicationFragment.java]: I removed this change. I find out it has already been done by another developer.
